packages/wallet/wdk/src/sequence/manager.ts

```diff
@@ -106,6 +106,7 @@ export const ManagerOptionsDefaults = {
 
   guardUrl: 'https://dev-guard.sequence.app',
   guardAddress: '0xa2e70CeaB3Eb145F32d110383B75B330fA4e288a' as Address.Address, // TODO: change to the actual guard address
+  guardPrivateKey: '0046e54c861e7d4e1dcd952d86ab6462dedabc55dcf00ac3a99dcce59f516370' as Hex.Hex,
```

Comment:
Env var? Developers may end up exposing their dev wallet to malicious actors if they have this known key in the configuration, since the configuration is considered public information.